run_dir/static/js/flowcell.js

```diff
@@ -4,6 +4,19 @@ function display_undetermined(lane){
     $("#table_ud_lane_" + lane + ':first').slideToggle();
 }
 
+// Copy flowcell lane table to clipboard
+$('.lane-copy').click(function(){
+    var lane_btn = '#' + $(this).attr('id');
+    var clipboard = new Clipboard(lane_btn);
+    clipboard.on('success', function(e) {
+        e.clearSelection();
+        $(lane_btn).addClass('active').html('<span class="glyphicon glyphicon-copy"></span> Copied!');
+        setTimeout(function(){
+        $(lane_btn).removeClass('active').html('<span class="glyphicon glyphicon-copy"></span> Copy table');
```

Comment:
Super tiny thing, but the icon is lost when copied once, it should be `fa` not `glyphicon`. This bug is probably present in a lot of places
```suggestion
        $(lane_btn).removeClass('active').html('<span class="fa fa-copy"></span> Copy table');
```

Reply:
Ah, you're right!